js/store.js

```diff
@@ -3,6 +3,7 @@
 
 // import your .js file here (e.g. import john from './modules/john.js')
 // ...
+import miksdumb from './modules/miksdumb.js';
```

Comment:
You only changed your GitHub username but you did not rename your module. So, you can keep your previous module file name.

```suggestion
import miksdumb from './modules/mikailashanin.js';
```